File: Haskell/beginner/1046.hs
-- https://judge.beecrowd.com/pt/problems/view/1046
-- Getting printf to print some awesome numbers
import Text.Printf

-- Main Function to do main things
main = do
    -- Requesting values and assign into variables
    usr_input <- getLine
    let [starting_hour, ending_hour] = map read (words usr_input) :: [Int]

    if starting_hour < ending_hour
        then do
            -- Showing results
            printf "O JOGO DUROU %d HORA(S)\n" (ending_hour - starting_hour)
        else do
            -- Showing results
            printf "O JOGO DUROU %d HORA(S)\n" (24 - starting_hour + ending_hour)

-- By Fernando Serra
-- https://github.com/fernandosserra

File: C/beginner/1046.c
/*https://judge.beecrowd.com/pt/problems/view/1046*/
/*Includes*/
#include <stdio.h>

/*Main*/
int main(){
    /*Variables*/
    int start_time, ending_time;

    /*Scanning values*/
    scanf("%d %d", &start_time, &ending_time);

    /*Testing conditions and showing results*/
    if (start_time < ending_time){
        printf("O JOGO DUROU %d HORA(S)\n", ending_time - start_time);
    } else if (start_time > ending_time){
        printf("O JOGO DUROU %d HORA(S)\n", 24 - start_time + ending_time);
    } else {
        printf("O JOGO DUROU 24 HORA(S)\n");
    }

    /*Ending Code*/
    return 0;
}


/*By Fernando Serra*/
/*https://github.com/fernandosserra*/